src/azure-cli/azure/cli/command_modules/synapse/commands.py

```diff
@@ -200,13 +211,35 @@ def get_custom_sdk(custom_module, client_factory):
         g.show_command('show', 'get')
         g.generic_update_command('update', custom_func_name='sqlpool_blob_auditing_policy_update')
 
+    # Management Plane Commands --Sql Ad-Admin
+    with self.command_group('synapse sql ad-admin', command_type=synapse_workspace_aad_admin_sdk,
+                            custom_command_type=get_custom_sdk('workspacesqlaadadmin',
+                                                               cf_synapse_client_workspace_aad_admins_factory),
+                            client_factory=cf_synapse_client_workspace_aad_admins_factory) as g:
+        g.show_command('show', 'get')
+        g.custom_command('create', 'create_workspace_sql_aad_admin', supports_no_wait=True)
+        g.generic_update_command('update', custom_func_name='update_workspace_sql_aad_admin',
+                                 setter_arg_name='aad_admin_info', supports_no_wait=True)
+        g.command('delete', 'delete')
```

Comment:
prefer adding confirmation=true for delete

Reply:
> prefer adding confirmation=true for delete

fixed.